src/display/node_stream.js

```diff
@@ -384,8 +389,14 @@ class PDFNodeStreamFsRangeReader extends BaseRangeReader {
   constructor(stream, start, end) {
     super(stream);
 
+    let path = decodeURI(this._url.path);
+
+    if (/^file:\/\/\/[a-zA-Z]:\//.test(this._url.href)) {
+      path = path.replace(/^\//, '');
+    }
+
     this._setReadableStream(
-      fs.createReadStream(decodeURI(this._url.path), { start, end: end - 1, }));
+      fs.createReadStream(decodeURI(path), { start, end: end - 1, }));
```

Comment:
You're now decoding the path twice because `path` is already decoded above. You should be able to remove the `decodeURI` call here.